Source/NSTableView.m

```diff
@@ -6839,18 +6786,20 @@ - (BOOL) _isCellEditableColumn: (NSInteger) columnIndex
       return [cell isEditable];
     }
 }
-
```

Comment:
There should be an empty line between the two methods.

Reply:
Done